src/vm/tests/costs.rs

```diff
@@ -876,9 +877,12 @@ fn test_all(use_mainnet: bool) {
     let baseline = test_tracked_costs("1", use_mainnet, StacksEpochId::Epoch20);
 
     for f in NativeFunctions::ALL.iter() {
-        let test = get_simple_test(f);
-        let cost = test_tracked_costs(test, use_mainnet, StacksEpochId::Epoch20);
-        assert!(cost.exceeds(&baseline));
+        // Note: The 2.05 test assumes Clarity1.
+        if f.get_version() == ClarityVersion::Clarity1 {
```

Comment:
Functions running in Stacks 2.0 can't be Clarity2, because the ClarityVersion is sometimes derived from the epoch, c.f. ```
impl ClarityVersion {
    pub fn latest() -> ClarityVersion {
        ClarityVersion::Clarity2
    }
    pub fn default_for_epoch(epoch_id: StacksEpochId) -> ClarityVersion {
        match epoch_id {
            StacksEpochId::Epoch10 => {
                warn!("Attempted to get default Clarity version for Epoch 1.0 where Clarity does not exist");
                ClarityVersion::Clarity1
            }
            StacksEpochId::Epoch20 => ClarityVersion::Clarity1,
            StacksEpochId::Epoch2_05 => ClarityVersion::Clarity1,
            StacksEpochId::Epoch21 => ClarityVersion::Clarity2,
        }
    }
}
```

Reply:
> Functions running in Stacks 2.0 can't be Clarity2, because the ClarityVersion is sometimes derived from the epoch, c.f.

I don't understand what you're saying here.  All Clarity functions that worked in Stacks 2.0 should continue to work in all future epochs.

Reply:
Clarity2 functions didn't exist in Stacks 2.0.

They are going to start existing in Stacks 2.1.